src/routes/ViewAllLogs/ViewAllLogs.js

```diff
@@ -396,6 +396,17 @@ class ViewAllLogs extends Component {
     sessionStorage.setItem(DATA_IMPORT_POSITION, null);
   }
 
+  handleResetAll = () => {
+    const { mutator: { query: { update } } } = this.props;
+
+    update({
+      filters: '',
+      sort: '-completedDate',
+      query: '',
+      qindex: '',
```

Comment:
Can this be moved to const and used here and in manifest for `query: { initialValue }`?